data/tabular/nomad_structure/meta.yaml

```diff
@@ -0,0 +1,126 @@
+---
+name: nomad-structure
+description: |-
+    A subset from NOMAD dataset, which is a database of DFT computed results of materials.
+    This subset consists of cif structures of around 0.5 million bulk stable materials and their geometric and structural information.
+    All materials in this dataset are modeled using Density Functional Theory using GGA functional.
+targets:
+    - id: density
+      description: Density of the material
+      units: Kg/m^3
+      type: continuous
+      names:
+          - noun: density of the material
+          - noun: density
+      uris:
+    - id: crystal_ststem
+      description: Geometric arrangement of atoms within a crystal
+      type: categorical
+      names:
+          - noun: crystal structure type
+          - noun: geometric arrangement of atoms within a crystal
+          - noun: arrangement of lattice points in space
+      uris:
+    - id: spacegroup
+      description: Spacegroup of the material
+      type: categorical
+      names:
+          - noun: spacegroup of the material
+          - noun: spacegroup
+      uris:
+    - id: pointgroup
+      description: Pointgroup of the material
+      type: categorical
+      names:
+          - noun: pointgroup of the material
+          - noun: pointgroup
+      uris:
+    - id: spacegroup_number
+      description: Spacegroup number of the material
+      type: categorical
+      names:
+          - noun: spacegroup number of the material
```

Comment:
```suggestion
```

in your templates, you already use "of the material". If you have it here, too, it will appear twice in some samples